server/libs/utils/utils.go

```diff
@@ -476,3 +476,11 @@ func IsNil(i interface{}) bool {
 	}
 	return false
 }
+
+// EscapeJSONString is used to escape special characters in JSON strings
+func EscapeJSONString(value string) string {
+	value = strings.ReplaceAll(value, `\`, `\\"`)
+	value = strings.ReplaceAll(value, `"`, `\"`)
+	value = strings.ReplaceAll(value, "\n", `\n`)
```

Comment:
we can use replacer instead of replace

https://medium.com/@vikram.ingawale91/golang-strings-replace-vs-strings-replacer-a7b2d2b71593

Reply:
> So, if you want to make a program that replaces more than two strings, use the strings NewReplacers Replace() method instead of strings Replace() method.

Reply:
Awesome, thanks for sharing.

Reply:
no need call strings.Replace or Replacer.  can  reference: https://github.com/Stenway/RSV-Challenge/blob/4948917321f9ae2cef0beb73be41dc959d07bd0b/Go/rsv.go#L244